test/src/specs/tx_pool/valid_since.rs

```diff
@@ -27,37 +26,24 @@ impl Spec for ValidSince {
     }
 
     fn modify_chain_spec(&self) -> Box<dyn Fn(&mut ChainSpec) -> ()> {
-        let cellbase_maturity = self.cellbase_maturity();
         Box::new(move |spec_config: &mut ChainSpec| {
-            spec_config.params.cellbase_maturity = cellbase_maturity;
+            spec_config.params.cellbase_maturity = 0;
```

Comment:
Remove code which test the _cellbase maturity_ in `ValidSince` spec, since the block number can't be estimated **accurately**.